task/02-numbers-tasks.js

```diff
@@ -72,8 +71,8 @@ function getAverage(value1, value2) {
  *   (-5,0) (10,-10) => 18.027756377319946
  *   (4, 2) (1, 6) => 5
  */
-function getDistanceBetweenPoints(x1, y1, x2, y2) {
-  throw new Error('Not implemented');
+function getDistanceBetweenPoints(x1, y1, x2, y2) {  
+  return Math.sqrt((Math.pow(Math.abs(x1 - x2), 2) + Math.pow(Math.abs(y1 - y2), 2)));
```

Comment:
when a number is squared, it's a positive number, so no need to get the abs value

Reply:
agree, removed it